orderapi/src/main/java/fi/hel/verkkokauppa/order/service/accounting/AccountingExportService.java

```diff
@@ -55,17 +59,17 @@ public void exportAccountingData(AccountingExportDataDto exportData) {
         AccountingSlipDto accountingSlipDto = new AccountingSlipTransformer().transformToDto(accountingSlip);
 
         String senderId = accountingSlipDto.getSenderId();
-        LocalDate timestamp = exportData.getTimestamp();
-        String filename = constructAccountingExportFileName(senderId, timestamp);
+        String filename = constructAccountingExportFileName(senderId, exportData.getTimestamp());
 
         export(exportData.getXml(), filename);
 
         markAsExported(exportData);
     }
 
     public String constructAccountingExportFileName(String senderId, LocalDate exportDataTimestamp) {
-        int year = exportDataTimestamp.getYear();
-        int count = exportDataRepository.countAllByExportedStartsWith(Integer.toString(year));
+        LocalDate start = LocalDate.of(exportDataTimestamp.getYear(), 1, 1);
```

Comment:
Can you rename variable to be startOfYear

Reply:
Sure, I just now pushed the fix for these variable names.